src/modules/fancyzones/lib/util.h

```diff
@@ -96,6 +96,32 @@ namespace FancyZonesUtils
             SRCCOPY);
     }
 
+    inline COLORREF HexToRGB(const std::wstring& hex)
+    {
+        std::wstring color = hex;
+        if (hex.size() == 7)
```

Comment:
let's use `trim` function instead of `size()` checking. trim space symbols and '#'.